aws/basic-eks-cluster/cloudformation/eks-workers.yaml.tmpl

```diff
@@ -110,15 +110,11 @@ Parameters:
   NodeAutoScalingGroupMinSize:
     Type: Number
     Description: Minimum size of Node Group ASG.
-    Default: 1
+    Default: "{{.NodeAutoScalingGroupDesiredSize}}"
   NodeAutoScalingGroupMaxSize:
     Type: Number
     Description: Maximum size of Node Group ASG.
-    Default: 4
-  NodeAutoScalingGroupDesiredSize:
-    Type: Number
-    Description: Desired size of Node Group ASG.
-    Default: 3
+    Default: "{{.NodeAutoScalingGroupDesiredSize}}"
```

Comment:
why are these changed, EKS doesn't let you have the same min/max in all region and NodeInstanceType, for some combinations EKS will error saying min/max can not be same, that's why I was adding +1 to the desiredSize for max

Reply:
also we are not utilizing the autoscale capacity if we set the same value for all

Reply:
let the user to decide how many nodes he wants

Reply:
that is the desired capacity we let people choose, also on EKS it makes sense to use autoscaling IMO, also the fact having same min/max doesnt work for some region/NodeInstanceType